autonomous_mobile_robot_2022/PointCloudData[] data